internal/awsservice/cloudwatchmetrics.go

```diff
@@ -66,12 +66,12 @@ func ValidateMetrics(t *testing.T, metricName, namespace string, dimensionsFilte
 
 }
 
-func ValidateSampleCount(t *testing.T, metricName, namespace string, dimensions []types.Dimension,
+func ValidateSampleCount(metricName, namespace string, dimensions []types.Dimension,
 	startTime time.Time, endTime time.Time,
 	lowerBoundInclusive int, upperBoundInclusive int, periodInSeconds int32) bool {
 	cwmClient, clientContext, err := GetCloudWatchMetricsClient()
 	if err != nil {
-		t.Fatalf("Error occurred while creating CloudWatch Logs SDK client: %v", err.Error())
```

Comment:
Is this okay?

Reply:
Yes. I intend to generalize the call which does not rely on test and only return error or value instead of bool https://github.com/aws/amazon-cloudwatch-agent-test/pull/103. Also all of the calls for `ValidateSampleCount` is [there](https://github.com/aws/amazon-cloudwatch-agent-test/pull/109/files#diff-b9fb7a5de31d3d0134ed0bd2138c43847925d56c93a78db9f988a7d85cfabbc6R101)